server/routes/bookVisit/selectVisitorsController.test.ts

```diff
@@ -20,64 +20,64 @@ const prisoner = TestData.prisonerInfoDto()
 const prison = TestData.prisonDto()
 
 const fakeDate = new Date('2024-05-02')
-const visitors = [
-  TestData.visitor({
-    visitorDisplayId: 1,
-    visitorId: 1000,
-    firstName: 'Visitor',
-    lastName: 'Age 20y',
-    dateOfBirth: '2004-04-01',
-  }),
-  TestData.visitor({
-    visitorDisplayId: 2,
-    visitorId: 2000,
-    firstName: 'Visitor',
-    lastName: 'Age 18y',
-    dateOfBirth: '2006-05-02', // 18 today
-  }),
-  TestData.visitor({
-    visitorDisplayId: 3,
-    visitorId: 3000,
-    firstName: 'Visitor',
-    lastName: 'Age 17y',
-    dateOfBirth: '2006-05-03', // 18 tomorrow
-  }),
-  TestData.visitor({
-    visitorDisplayId: 4,
-    visitorId: 4000,
-    firstName: 'Visitor',
-    lastName: 'Age 16y',
-    dateOfBirth: '2008-05-02', // 16 today
-  }),
-  TestData.visitor({
-    visitorDisplayId: 5,
-    visitorId: 5000,
-    firstName: 'Visitor',
-    lastName: 'Age 15y',
-    dateOfBirth: '2008-05-03', // 16 tomorrow
-  }),
-  TestData.visitor({
-    visitorDisplayId: 6,
-    visitorId: 6000,
-    firstName: 'Visitor',
-    lastName: 'Age 10y',
-    dateOfBirth: '2014-05-02',
-  }),
-  TestData.visitor({
-    visitorDisplayId: 7,
-    visitorId: 7000,
-    firstName: 'Visitor',
-    lastName: 'Age 1y',
-    dateOfBirth: '2023-05-02',
-  }),
-  TestData.visitor({
-    visitorDisplayId: 8,
-    visitorId: 8000,
-    firstName: 'Visitor',
-    lastName: 'Age 4m',
-    dateOfBirth: '2024-01-02',
-  }),
-]
+
+const visitor1 = TestData.visitor({
```

Comment:
Just renamed all these so they can be referred to by their ID numbers - rather than mixing that with accessing them from a zero-indexed array. Was getting myself confused 😂

Reply:
Is their age all that changes? Wonder if that could be in the var name rather an index? Might be easier to understand what visitors are used for testing rather than referring back to see what age visitor3 is for example?

Reply:
Good suggestion - I'll have a look at that on a subsequent PR...